dotenv/cut_go117.go

```diff
@@ -0,0 +1,13 @@
+//go:build !go1.18
+// +build !go1.18
+
+package dotenv
+
+import "bytes"
+
+func bytesCut(s, sep []byte) (before, after []byte, found bool) {
+	if i := bytes.Index(s, sep); i >= 0 {
+		return s[:i], s[i+len(sep):], true
+	}
+	return s, nil, false
+}
```

Comment:
Do we really need this as Golang `1.19` has been released and Go Team only support 2 versions of Golang, we may assume that `1.18` is the default minimal version now, wdyt?
I opened a [PR](https://github.com/compose-spec/compose-go/pull/309) to update the CI workflow

Reply:
This is now gone and using `bytes.Cut` directly after merging in #309. 🙂